### ISSUE TITLE
Action Bars Do Not Display

### ISSUE THREAD

**Issue by lock720** [tullamods/Dominos#338]
When logging in with just Dominos enabled no action bars are shown on screen and an error message is shown in chat:

Failed to load ActionBars
Interface\FrameXML\ActionButton.lua:486: assertion failed!

**Comment by Tuller** [comment]
Make sure that the addon is installed to the right directory (WoW Folder\_retail_\Interface\Addons). You may need to update/reinstall the twitch client if you're using that.

**Comment by Daxx101** [comment]
Hi hi, I am also having this problem. the actual action bars arent loading at all. the exp bar, azerite bar, stealth, pet, bags and menu icons are working. when i hit configure bars there arent any of the numbered action bars appearing at all. I tried reinstalling the addon without deleting my preferences through the twitch app but no luck. I hope I don't have to reinstall the whole thing along with my preferences D:  I also checked it was installedi n the right place and the error I am getting is what the same "
 Interface\FrameXML\ActionButton.lua:486: assertion failed!"
